src/ore/src/vec.rs

```diff
@@ -48,10 +48,12 @@ pub trait Vector<T> {
 }
 
 impl<T> Vector<T> for Vec<T> {
+    #[inline]
```

Comment:
Why did you decide to mark these single-expression functions `#[inline]`, rather than `#[inline(always)]`?